src/components/Root/KeystoneCustom.module.scss

```diff
@@ -0,0 +1,5 @@
+/* Onboard.js custom styles */
+
+:global(.ReactModal__Overlay) {
+  z-index: 2002 !important;
+}
```

Comment:
Is the modal being hidden behind something else? Why is it specific to the Keystone wallet?

Reply:
Hey @katspaugh , thanks for your review. 
That's because the style in [OnboardCustom.module.scss](https://github.com/gnosis/safe-react/blob/dev/src/components/Root/OnboardCustom.module.scss#L3) improve the z-index of Onboard's modal which blocks Keystone's modal.